<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DownloadManager</class>
 <widget class="QMainWindow" name="DownloadManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>等距更纱黑体 SC</family>
    <pointsize>12</pointsize>
    <weight>75</weight>
    <italic>true</italic>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>DownloadManager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_URL">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>URL</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_URL">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>600</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Path">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Path</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Path">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>150</y>
      <width>600</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Start">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>240</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等距更纱黑体 SC</family>
     </font>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Pause">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>320</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Name">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>240</y>
      <width>500</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>File Name: </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Size">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>320</y>
      <width>500</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>File Size: </string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
